cabal-testsuite/PackageTests/BuildToolDependsInternalMissing/setup.out

```diff
@@ -1,3 +1,3 @@
 # Setup configure
 Configuring build-tool-depends-missing-0.1.0.0...
-Error: setup: The package depends on a missing internal executable: 
+Error: setup: The package depends on a missing internal executable: build-tool-depends-missing:hello-world
```

Comment:
Introducint `MissingInternalExe` highlighted a bug, I corrected it and accepted the new test output.